SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -2471,6 +2484,10 @@ void CFVMFlowSolverBase<V, FlowRegime>::Friction_Forces(const CGeometry* geometr
     su2double ForceViscous[MAXNDIM] = {0.0}, MomentViscous[MAXNDIM] = {0.0};
     su2double MomentX_Force[MAXNDIM] = {0.0}, MomentY_Force[MAXNDIM] = {0.0}, MomentZ_Force[MAXNDIM] = {0.0};
 
+    /* --- check if wall functions are used --- */
+   
+    const bool wallfunctions = (config->GetWallFunction_Treatment(Marker_Tag) != NO_WALL_FUNCTION);
```

Comment:
While you are at it: can you make this an enum class following #1258 and naming the enum e.g. `WALL_FUNCTIONS`